### PR TITLE
[formrecognizer] Fix test assert for changing description value

### PR DESCRIPTION
**Pull request by catalinaperalta**
# Description

Operations may or may not have a description, this PR fixes the check for those test assertions.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by catalinaperalta** [comment]
The ci pipeline passed, running the test pipeline just to see if the issues with the list operations tests are resolved. There may still be some flakiness for other tests.

**Review by kristapratico — APPROVED** [review]

File: scripts/devops_tasks/test_run_samples.py [hunk 1]
```diff
@@ -130,6 +130,8 @@
     "azure-ai-formrecognizer": [
         "sample_manage_custom_models.py",
         "sample_manage_custom_models_async.py",
+        "sample_copy_model.py",
+        "sample_copy_model_async.py",
```
Comment (catalinaperalta) [inline scripts/devops_tasks/test_run_samples.py:134]: Tracking issue: https://github.com/Azure/azure-sdk-for-python/issues/22185

**Review by kristapratico — APPROVED** [review]

**Pull request merged**